samples/constitutions/default/actions.js

```diff
@@ -996,27 +996,26 @@ const actions = new Map([
       function (args) {
         checkType(args.security_policy, "string", "security_policy");
         checkType(args.host_data, "string", "host_data");
-      },
-      function (args, proposalId) {
-        const host_data = ccf.strToBuf(args.host_data);
-        const security_policy = ccf.jsonCompatibleToBuf(args.security_policy);
 
+        // If optional security policy is specified, make sure its 
```

Comment:
Moving this check to new "validate" part of proposal action as this can be entirely checked on proposal creation rather than execution as this doesn't interact with the KV.